kit/azure/bootstrap/main.tf

```diff
@@ -4,14 +4,17 @@ data "azurerm_management_group" "root" {
   name = var.aad_tenant_id
 }
 
+data "azurerm_subscription" "current" {
+}
+
 // we put the terraform_state part into its own module as that simplifies making it optional
 module "terraform_state" {
   count = var.terraform_state_storage != null ? 1 : 0
 
-  source                    = "./terraform-state"
-  location                  = var.terraform_state_storage.location
-  file_path                 = var.file_path
-  resources_cloudfoundation = var.resources_cloudfoundation
```

Comment:
the 'resources_cloudfoundation' we also use as an dependency in the activity-log kit. Lgtm, but we should test it.

Reply:
thanks for the pointer, I will address this in #47